cmd/gateway-main.go

```diff
@@ -286,7 +292,7 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 		}
 
 		// Print gateway startup message.
-		printGatewayStartupMessage(getAPIEndpoints(gatewayAddr), gatewayName)
+		printGatewayStartupMessage(getAPIEndpoints(gatewayAddr), gatewayName, anonFlag)
```

Comment:
Perhaps make a global setting rather than taking an Input here. It is just not worth expanding the scope of `printGatewayStartupMessage` function.

Reply:
> Perhaps make a global setting rather than taking an Input here. It is just not worth expanding the scope of `printGatewayStartupMessage` function.

I am more worried to add more global variables here

Reply:
yeah but there is no value in adding these flags in this manner.